changelogs/unreleased/ghqa-321-segfault-with-clang-16-asan.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Fixed a crash that could happen when Tarantool is compiled by `clang`
+  version 16 with enabled AddressSanitizer (tarantool/tarantool-qa#321).
```

Comment:
It fails with clang 15 and above.

Reply:
Changed to "clang version 15 and above".